bedrock/base/templates/includes/site-footer.html

```diff
@@ -58,7 +58,12 @@
     <nav class="secondary">
         <div class="small-links">
           <ul>
-            <li><a rel="nofollow" href="{{ url('privacy') }}" data-link-type="footer" data-link-name="Privacy">{{ _('Privacy') }}</a></li>
+            {% if LANG == 'en-US' %} 
+              {% set privacy_text = _('Website Privacy Notice') %}
```

Comment:
If the new text here is `Websites Privacy Notice`, then I would kind of expect the link go to `/privacy/websites/`. This feels like a bit of an odd choice of wording, what's the reasoning behind it?

Reply:
Both `Mozilla Privacy Center` and `Mozilla Privacy Policy` seem like better suggestions (which we're previously floated in the bug)

Reply:
You're calling out something I overlooked. In https://bugzilla.mozilla.org/show_bug.cgi?id=1367183#c10 Brian suggests we change the link to /websites. I'll fixup.

Reply:
fixed upped.